WatchDogIntelliJPlugin/WatchDog/src/nl/tudelft/watchdog/intellij/logic/InitializationManager.java

```diff
@@ -22,7 +23,7 @@
     private static final int USER_ACTIVITY_TIMEOUT = 16000;
 
     /** The singleton instance of the interval manager. */
-    private static volatile InitializationManager instance = null;
+    private static volatile HashMap<String, InitializationManager> instances = new HashMap<String, InitializationManager>();
```

Comment:
Why is this in the IntelliJ version? Couldn't it be in the core, and Eclipse implicitly using it?